Hexastore.Test/RocksFixture.cs

```diff
@@ -20,15 +20,16 @@ public class RocksFixture : IDisposable
         public readonly IStoreProcesor StoreProcessor;
         public readonly IStoreOperationFactory StoreOperationFactory;
         public readonly string SetId;
+        public readonly string TestDirectory;
 
         public RocksFixture()
         {
-            const string testDirectory = "./testdata";
-            if (Directory.Exists(testDirectory)) {
-                Directory.Delete(testDirectory, true);
+            TestDirectory = $"./{Guid.NewGuid()}";
```

Comment:
You can copy TestFolder from PerfConsole and use it here, it does the same thing.  https://github.com/angshuman/hexadb/blob/master/Hexastore.PerfConsole/TestFolder.cs

 I think all tests should set up their own rocksdb and folder instead of using a shared one across tests, which could impact others, but that would be a larger change.